src/components/reformed.js

```diff
@@ -89,10 +91,10 @@ const makeWrapper = WrappedComponent => {
       }
     }
 
-    bindInput(name) {
+    bindInput(name, isFile = false) {
```

Comment:
Foreslått løsning vil ikke fungere. Grunnen til at jeg gjorde slikt er at `<input name={name} type="file" />` gir ikke tillatelse og sette `value`. Det kommer av sikkerthetsmessige grunner.
https://stackoverflow.com/questions/1696877/how-to-set-a-value-to-a-file-input-in-html